inscriptions/ISic000840.xml

```diff
@@ -186,21 +188,36 @@
             <graphic n="print" url="ISic000840.jpg" height="3680px" width="5520px">
                 <desc>I.Sicily with the permission of the Assessorato Regionale dei Beni Culturali e dell’Identità Siciliana - Dipartimento dei Beni Culturali e dell’Identità Siciliana</desc>
             </graphic>
-         </surface>
+        </surface>
     </facsimile>
     <text>
         <body>
             <div type="edition" xml:space="preserve" xml:lang="grc" resp="#VM">
                 <ab>
-                    <lb n="1"/><expan><abbr>Θ</abbr><ex>εοῖς</ex></expan> <expan><abbr>Κ</abbr><ex>αταχθονίοις</ex></expan>
-                    <lb n="2"/>Ἀφροδισιὰς
-                    <lb n="3"/>Διονυσίου
-                    <lb n="4"/>χαῖ<supplied reason="lost">ρε</supplied> Φιλίστ<supplied reason="lost">η</supplied>
-                    <lb n="5"/>μήτηρ <supplied reason="lost">ἐποίησε</supplied>
-                    <lb n="6"/>ζησ<supplied reason="omitted">ά</supplied>σ<supplied reason="lost">ῃ</supplied><gap reason="lost" extent="unknown" unit="character"/>
+                    <lb n="1"/><w n="5"><expan><abbr>Θ</abbr><ex>εοῖς</ex></expan></w> <w n="10"><expan><abbr>Κ</abbr><ex>αταχθονίοις</ex></expan></w>
+                    <lb n="2"/><w n="15">Ἀφροδισιὰς</w>
+                    <lb n="3"/><w n="20">Διονυσίου</w>
+                    <lb n="4"/><w n="25">χαῖ<supplied reason="lost">ρε</supplied></w> <w n="30">Φιλίστ<supplied reason="lost">η</supplied></w>
+                    <lb n="5"/><w n="35">μήτηρ</w> <w n="40"><supplied reason="lost">ἐποίησε</supplied></w>
+                    <lb n="6"/><w n="45">ζησ<supplied reason="omitted">ά</supplied>σ <supplied reason="lost">ῃ</supplied></w> <gap reason="lost" extent="unknown" unit="character"/>
```

Comment:
again, we've got a space inserted before the second supplied element, although the word correctly identified.